main.py

```diff
@@ -1,5 +1,30 @@
 import init_django_orm  # noqa: F401
 
+from db.models import Genre, Actor
+
 
 def main():
-    pass
+
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
+
+    Actor.objects.create(first_name="George", last_name="Klooney")
```

Comment:
it`s correct, but, i think, will be better, create write in tables use loop, and write data from? some list or something else

Reply:
here, broken DRY